apps/microtvm/arduino/host_driven/src/model.c

```diff
@@ -0,0 +1,73 @@
+#ifndef TVM_IMPLEMENTATION_ARDUINO
+#define TVM_IMPLEMENTATION_ARDUINO
+
+#include "model.h"
```

Comment:
I want to keep the name `model.c` in the example_project, since I think it's most intuitive for users to `#include "src/model.c"`. For the `host_driven` project though, I agree it can be changed. Fixed.